lib/rot.rb

```diff
@@ -0,0 +1,26 @@
+
+class Rot
+    ALPHABET = ("a".."z").to_a + ("A".."Z").to_a
+    ALPHABET.freeze
```

Comment:
@djordje997 you can freeze it on the same line.

```suggestion
    ALPHABET = (("a".."z").to_a + ("A".."Z").to_a).freeze
```